.github/workflows/directory_writer.yml

```
@@ -10,20 +10,24 @@ on:
   #        │  │ │ │ │
   #        │  │ │ │ │
   #        *  * * * *
-  - cron: '0 0 * * *'
-  workflow_dispatch:
+  #- cron: '0 0 * * *'
+  #workflow_dispatch:
```

Please add `workflow_dispatch` again. This will allow us to run it whenever we want.
```suggestion
  workflow_dispatch:
```